consumers/hidden-test-consumer/src/hidden_test_consumer/process.py

```
@@ -52,7 +58,9 @@
         self.grpc_server = grpc_server
 
         self.download_dir = os.environ.get("DOWNLOAD_DIR", "/tmp")
```

Ruff (0.8.2)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)